src/tests/Microsoft.PowerFx.Connectors.Tests/BasicRestTests.cs

```diff
@@ -25,6 +29,56 @@ private static void AssertLog(LoggingTestServer testConnector, string expectedLo
 
             testConnector._log.Clear();
         }
+        
+        private const string Response = @"[{""date"":""2022-06-09T17:43:33.6483791+02:00"",""temperatureC"":-15,""temperatureF"":6,""summary"":""Bracing"",""index"":121}]";
```

Comment:
> @"[{""date"":""2022-06-09T17:43:33.6483791+02:00"",""temperatureC"":-15,""temperatureF"":6,""summary"":""Bracing"",""index"":121}]"; [](http://example.com/codeflow?start=39&length=133)

Can we simplify the return value here. 
These tests are testing parameter calls; so complex return values are just more complexity. 

We could have other tests for the complex return values.

Reply:
In fact, maybe just remove the response check altogether - and handle that in a separate test.

Reply:
Makes sense